### ISSUE TITLE
[CMake Language Support] Error: Could not resolve dotnet path!

### ISSUE THREAD

**Issue by Dragon1573** [dotnet/vscode-dotnet-runtime#627]
**Repro steps:**

1.

**Extension Version:** 1.0.0
**Error Message:** .NET Acquisition Failed: Installation failed: Error: Command failed: "/anonymous/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/install scripts/dotnet-install.sh" -InstallDir "/anonymous/.vscode-server/data/User/globalStorage/ms-dotnettools.vscode-dotnet-runtime/.dotnet/6.0.8" -Version 6.0.8 -Runtime dotnet
/anonymous/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/install scripts/dotnet-install.sh: 7: Syntax error: newline unexpected

**Error Stack:** Error: .NET Acquisition Failed: Installation failed: Error: Command failed: "/anonymous/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/install scripts/dotnet-install.sh" -InstallDir "/anonymous/.vscode-server/data/User/globalStorage/ms-dotnettools.vscode-dotnet-runtime/.dotnet/6.0.8" -Version 6.0.8 -Runtime dotnet
/anonymous/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/install scripts/dotnet-install.sh: 7: Syntax error: newline unexpected

	at /anonymous/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/extension.js:2:13750

**Comment by Dragon1573** [comment]
I'm currently using Ubuntu 22.04 LTS for WSL 2. I've installed .NET 6.0 in the instance as follows:

<details>
<summary>Terminal Logs</summary>

```text
Welcome to fish, the friendly interactive shell
Type help for instructions on how to use fish
➜  CMake neofetch 
            .-/+oossssoo+/-.               OS: Ubuntu 22.04.1 LTS on Windows 10 x86_64 
        `:+ssssssssssssssssss+:`           Kernel: Linux 5.15.68.1-microsoft-standard-WSL2 
      -+ssssssssssssssssssyyssss+-         Packages: 1402 (dpkg) 
    .ossssssssssssssssssdMMMNysssso.       Shell: fish 3.3.1 
   /ssssssssssshdmmNNmmyNMMMMhssssss/      Theme: Breeze [GTK2/3] 
  +ssssssssshmydMMMMMMMNddddyssssssss+     Icons: breeze [GTK2/3] 
 /sssssssshNMMMyhhyyyyhmNMMMNhssssssss/    Terminal: vscode 
.ssssssssdMMMNhsssssssssshNMMMdssssssss.   CPU: 11th Gen Intel i7-1165G7 (8) @ 2.8GHz 
+sssshhhyNMMNyssssssssssssyNMMMysssssss+   GPU: 57fe:00:00.0 Microsoft Corporation Device 008e 
ossyNMMMNyMMhsssssssssssssshmmmhssssssso   Memory: 674MiB / 7843MiB (8%) 
ossyNMMMNyMMhsssssssssssssshmmmhssssssso   Disk (c): 90G / 120G (75%) 
+sssshhhyNMMNyssssssssssssyNMMMysssssss+   Disk (d): 48G / 821G (6%) 
.ssssssssdMMMNhsssssssssshNMMMdssssssss.   Font: Noto Sans, 10 [GTK2/3] 
 /sssssssshNMMMyhhyyyyhdNMMMNhssssssss/    Public IP: *** MASKED *** 
  +sssssssssdmydMMMMMMMMddddyssssssss+
   /ssssssssssshdmNNNNmyNMMMMhssssss/                                      
    .ossssssssssssssssssdMMMNysssso.                                       
      -+sssssssssssssssssyyyssss+-
        `:+ssssssssssssssssss+:`
            .-/+oossssoo+/-.

➜  CMake dotnet --info 
.NET SDK (reflecting any global.json):
 Version:   6.0.110
 Commit:    ce0a42998a

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  22.04
 OS Platform: Linux
 RID:         ubuntu.22.04-x64
 Base Path:   /usr/lib/dotnet/dotnet6-6.0.110/sdk/6.0.110/

global.json file:
  Not found

Host:
  Version:      6.0.10
  Architecture: x64
  Commit:       5a400c212a

.NET SDKs installed:
  6.0.110 [/usr/lib/dotnet/dotnet6-6.0.110/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.10 [/usr/lib/dotnet/dotnet6-6.0.110/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.10 [/usr/lib/dotnet/dotnet6-6.0.110/shared/Microsoft.NETCore.App]

Download .NET:
  https://aka.ms/dotnet-download

Learn about .NET Runtimes and SDKs:
  https://aka.ms/dotnet/runtimes-sdk-info

➜  CMake dotnet --list-sdks 
6.0.110 [/usr/lib/dotnet/dotnet6-6.0.110/sdk]

➜  CMake dotnet --list-runtimes 
Microsoft.AspNetCore.App 6.0.10 [/usr/lib/dotnet/dotnet6-6.0.110/shared/Microsoft.AspNetCore.App]
Microsoft.NETCore.App 6.0.10 [/usr/lib/dotnet/dotnet6-6.0.110/shared/Microsoft.NETCore.App]
```

</details>

I configured the extension to use my pre-installed version of .NET 6 as follows:

<details>
<summary><code>/root/.vscode-server/data/Machine/settings.json</code></summary>

```json
{
    "java.configuration.runtimes": [
        {
            "name": "JavaSE-17",
            "default": true,
            "path": "/usr/lib/jvm/java-17-openjdk-amd64",
            "javadoc": "/usr/lib/jvm/openjdk-17/src.zip"
        }
    ],
    "dotnetAcquisitionExtension.existingDotnetPath": [
        {
            "extensionId": "josetr.cmake-language-support-vscode",
            "path": "/usr/lib/dotnet/dotnet6-6.0.110/dotnet"
        }
    ]
}
```

</details>

But whenever I launch VSCode, it'll prompt an error for be bottom right.

![image](https://user-images.githubusercontent.com/49941141/195767567-efdf0b24-7bc3-4231-90bb-48eeccb401bb.png)

![image](https://user-images.githubusercontent.com/49941141/195767631-763d3ce6-d783-4709-b12f-f1f1ecb75931.png)

WHAT Linux libraries should I install? Is there any docs mentioned this?

**Comment by nagilson** [comment]
Heya, thanks for the detailed issue. This is a dupe of https://github.com/dotnet/vscode-dotnet-runtime/issues/159 and will be fixed in the 1.6.1 release. Closing, but feel free to ask any concerns over there.
